src/components/toast/Toast.tsx

```diff
@@ -65,27 +67,40 @@ export class Toast extends React.Component<IToastProps, {}> {
     }
 
     render() {
-        const classes = classNames('toast', {
-            'mod-success': this.props.type !== ToastType.Warning && this.props.type !== ToastType.Error,
-            'mod-warning': this.props.type === ToastType.Warning,
-            'mod-error': this.props.type === ToastType.Error,
-            'mod-animated': _.isUndefined(this.props.animate) || this.props.animate === true,
-        });
+        const classes = classNames(
+            'toast',
+            {
+                'mod-success': this.props.type === ToastType.Success,
+                'mod-warning': this.props.type === ToastType.Warning,
+                'mod-error': this.props.type === ToastType.Error,
+                'mod-animated': _.isUndefined(this.props.animate) || this.props.animate === true,
+            },
+            this.props.className,
+        );
+
+        const closeButton = this.props.dismissible ?
+            <span className='toast-close' onClick={() => this.close()}>

```

Comment:
can we use a button instead of a div with a onClick, now the Button can have a Svg as children

Reply:
Probably, I'll try refactoring this also

Reply:
```
? (
 // your component
)
: null;
```